src/app/components/board.jsx

```diff
@@ -1,16 +1,16 @@
 'use client'
 
-import React, { useState } from 'react';
+import { useState } from 'react';
 import Cell from './cell'
 
 const board = [...Array(9)]
 
 export default function Board() {
   
   const [size, setSize] = useState(9)
-  
+
   return (
-    <div className="flex flex-row">
+    <div className="flex flex-row border border-1 border-slate-500 justify-center items-center w-screen h-screen">
```

Comment:
you're readding border and border-1 which I've eliminated it caused weird lines in UI